worlds/sc2/items.py

```diff
@@ -919,6 +922,9 @@ def get_full_item_list():
         ItemData(624 + SC2WOL_ITEM_ID_OFFSET, TerranItemType.Laboratory, 13, SC2Race.TERRAN, origin={"ext"}),
     item_names.MERCENARY_MUNITIONS:
         ItemData(625 + SC2WOL_ITEM_ID_OFFSET, TerranItemType.Laboratory, 14, SC2Race.TERRAN, origin={"ext"}),
+    item_names.DEVASTATOR_TURRET_CONCUSSIVE_GRENADES:
+        ItemData(396 + SC2WOL_ITEM_ID_OFFSET, TerranItemType.Laboratory, 15, SC2Race.TERRAN,
+                 parent_item=item_names.DEVASTATOR_TURRET, origin={"ext"}),
```

Comment:
Should be an armory upgrade, not Laboratory one (like missile turret upgrades are)

And the item ID doesn't follow

Reply:
Put it on index armory 6 index 30 before realizing the indices only go up to 29. Looks like we're out of space in all 6 armory categories and now need a seventh `:/`